### PR TITLE
Christopher foster stretch

### PR DESCRIPTION
**Pull request by catslanderer**

### CODE REVIEW

File: data/models/dishes-model.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+const addDish = dish => {
+  return db('dishes').insert(dish);
+};
+
+const getDish = id => {
+  return db('dishes')
+    .leftJoin('recipes', 'dishes.id', 'recipes.dishes_id')
+    .select('dishes.id', 'dishes.name as dish', 'recipes.name as recipe')
+    .where({ 'dishes.id': id });
```
Comment (DaftBeowulf) [inline data/models/dishes-model.js:15]: interesting, this exact same request will now return an array of recipes per single dish object as I mentioned would be preferred in your last  branch, but the code here is the same--did you change something in your seeds or migrations in between these two PRs?

File: data/models/recipes-model.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+const addRecipe = recipe => {
+  return db('recipes').insert(recipe);
+};
+
+const getRecipe = async id => {
+  return db('recipes')
+    .leftJoin('recipes_ings', 'recipes_ings.recipes_id', 'recipes.id')
+    .leftJoin('ings', 'recipes_ings.ings_id', 'ings.id')
+    .join('dishes', 'dishes.id', 'recipes.dishes_id')
```
Comment (DaftBeowulf) [inline data/models/recipes-model.js:17]: very complex multi-join here, great job!

**Review by DaftBeowulf — COMMENTED** [review]
Good job with even more stretch for this assignment Chris, all your responses are returning data in the desired form including the deceptively complex getRecipeById :100:

File: data/models/dishes-model.js (hunk 1 above)
Comment (catslanderer) [reply data/models/dishes-model.js:15]: You may have already figured this out, but I use JS in the endpoint to take the SELECT data and turn it into the form the specifications asked for.

**Pull request merged**